### PR TITLE
boards: stk3x00: add docs

### PR DESCRIPTION
**Pull request by basilfx**
### Contribution description
Add documentation for the STK3600 and STK3700.

### Testing procedure
Doxygen output is OK.

### Issues/PRs references
#9881 
#9880

### CODE REVIEW

File: boards/stk3600/doc.txt [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+| PD4  | 12  | 11  | PB11 |
+| PD3  | 10  | 9   | PC5  |
+| PD2  | 8   | 7   | PC4  |
+| PD1  | 6   | 5   | PC3  |
+| PD0  | 4   | 3   | PC0  |
+| VMCU | 2   | 1   | GND  |
+
+**Note**: not all starter kits by Silicon Labs share the same pinout!
+
+**Note:** some pins are connected to the board controller, when enabled!
```
Comment (smlng) [inline boards/stk3600/doc.txt]: be consistent where to the colon, compare note here and above

File: boards/stk3700/doc.txt [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+| PD6  | 16  | 15  | PC6  |
+| PD5  | 14  | 13  | PB12 |
+| PD4  | 12  | 11  | PB11 |
+| PD3  | 10  | 9   | PC5  |
+| PD2  | 8   | 7   | PC4  |
+| PD1  | 6   | 5   | PC3  |
+| PD0  | 4   | 3   | PC0  |
+| VMCU | 2   | 1   | GND  |
+
+**Note**: not all starter kits by Silicon Labs share the same pinout!
```
Comment (smlng) [inline boards/stk3700/doc.txt]: move colon inside to match other notes

File: boards/stk3600/doc.txt [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+The starter kit is equipped with an Advanced Energy Monitor. This allows you to actively measure the power consumption of your hardware and code, in real-time.
+
+## Hardware
+
+### MCU
+| MCU             | EFM32LG990F256                                       |
+|-----------------|------------------------------------------------------|
+| Family          | ARM Cortex-M3                                        |
+| Vendor          | Silicon Labs                                         |
+| Vendor Family   | EFM32 Leoppard Gecko                                 |
```
Comment (smlng) [inline boards/stk3600/doc.txt]: double `p`?

**Comment by basilfx** [comment]
@smlng I have fixed your comments and rebased.

**Comment by basilfx** [comment]
@smlng I know, and I am happy to change that. But the discussion about line length for documentation is not yet concluded (see [here](https://github.com/RIOT-OS/RIOT/pull/9881#issuecomment-432594405)). So I am in favor of merging this as-is, and fix this in a follow-up. Better than no documentation.

**Comment by miri64** [comment]
Not much to say on this... so I withdraw my review.

**Comment by PeterKietzmann** [comment]
@basilfx I think the discussion in #9881 was never 'officially' resolved (except for the coding conventions) but I see stronger opinions in keeping the line width short even in documentation. Personally I don't have a strong opinion but I'd suggest to simply limit the length here, so we can finally merge this piece of documentation.

**Comment by jia200x** [comment]
ping @basilfx

**Comment by basilfx** [comment]
I close this in favor of the newer #13117 and #13116.

**Pull request closed without merging**